packages/rest-typings/src/v1/webdav.ts

```diff
@@ -0,0 +1,9 @@
+import type { IWebdavAccount } from "@rocket.chat/core-typings";
+
+export type WebdavEndpoints = {
+	'webdav.getMyAccounts': {
+		GET: () => {
+			accounts: Pick<IWebdavAccount, '_id' | 'username' | 'serverURL' | 'name'>[];
```

Comment:
what about create a type to hold this variation? does make any sense sharing this on other place or not?

Reply:
idk actually, there is no other place using it.. 🤷‍♂️